test/unittest/unittest_nntrainer_modelfile.cpp

```diff
@@ -95,13 +115,23 @@ static int ALLFAIL = LOADFAIL | INITFAIL;
 
 using I = IniSection;
 
-/// @note each line contains 3 test, so this should be counted * 3
+/// @note each line contains 2 (positive or negative test) + 1 negative test.
+/// if, there are 6 positive tests and 9 negative tests
+/// which sums up to 6 * 2 = 12 positive tests and 9 * 2 + 6 + 9 = 33 negative
+/// tests
 // clang-format off
 INSTANTIATE_TEST_CASE_P(
   nntrainerIniAutoTests, nntrainerIniTest, ::testing::Values(
   /**< positive: basic valid scenarios */
     mkIniTc("basic_p", {nw_adam, input, out}, SUCCESS),
+    mkIniTc("basic2_p", {nw_sgd, input, out}, SUCCESS),
+    mkIniTc("basic_act_p", {nw_sgd, input + "-Activation", act_relu, out }, SUCCESS),
+    mkIniTc("basic_bn_p", {nw_sgd, input + "-Activation", batch_normal, act_relu, out }, SUCCESS),
+    // #390
```

Comment:
Lets use DISABLED_ prefix than commenting these tests.
Also, do you want to add this PR first or make these fixes and then merge this patch?

Reply:
There is no way to change the test name for <= 1.7.0(gtest) for parameterized test so we can't do that.

I think it is better to merge this first and uncomment each tests to check if particular issue has been fixed since there are plenty

Reply:
ok.